ports/esp32/machine_sdcard.c

```diff
@@ -193,11 +198,16 @@ STATIC mp_obj_t machine_sdcard_make_new(const mp_obj_type_t *type, size_t n_args
     self->flags = 0;
     // Note that these defaults are macros that expand to structure
     // constants so we can't directly assign them to fields.
+    
+    int freq = arg_vals[ARG_bitrate].u_int;
+    
     if (is_spi) {
         sdmmc_host_t _temp_host = SDSPI_HOST_DEFAULT();
         self->host = _temp_host;
     } else {
         sdmmc_host_t _temp_host = SDMMC_HOST_DEFAULT();
+        // Use SDMMC_FREQ_HIGHSPEED = 40MHz, about 30% read/write speed improve over SDMMC_FREQ_DEFAULT = 20MHz
+        _temp_host.max_freq_khz = freq;
```

Comment:
Now the comment is not right here anymore. Maybe just drop it and keep the info in the commit message only?